worf/views/list.py

```diff
@@ -181,8 +181,11 @@ def paginated_results(self):
         queryset = self.get_processed_queryset()
         request = self.request
 
-        if settings.DEBUG:
-            self.query = str(queryset.query)
+        if settings.WORF_DEBUG:
+            try:
+                self.query = str(queryset.query)
+            except EmptyResultSet:
+                self.query = None
```

Comment:
You might have seen this exception when you use filters that create an impossible query, I think outside of debug mode it just yields `[]` but when debug mode is on it would fail here trying to output the sql for the debug dict.